test/sql/copy/csv/test_insert_into_hive.test

```diff
@@ -0,0 +1,75 @@
+# name: test/sql/copy/csv/test_insert_into_hive.test
+# description: Test insert of real-Hive-partitioned CSV data
+# group: [csv]
+
+statement ok
+PRAGMA enable_verification
+
+statement ok
+create table test_insert_hive (id int, value varchar, date date, part varchar);
+
+# successful insert when partition data and csv data match destination table columns
+statement ok
+insert into test_insert_hive by name select * from read_csv('data/csv/hive-partitioning/simple/*/*/test.csv');
+
+# query IIII
```

Comment:
Can we uncomment these tests?

Reply:
I had some uncommitted work on my local... apologies. Pushed and merged master now.